src/index.html

```diff
@@ -5,9 +5,291 @@
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="header__action">
+          <a href="tel:+1234555-55-55" class="header__link">Call to order</a>
+        </div>
+      </div>
+
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--active">
+            Home
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              About Us
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gallery
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Consultation
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Testimonials
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Contacts
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="section">
+        <div class="info">
+          <h1 class="info__heading">
+            Miami
+            <span class="info__selected-text info__selected-text--bold">
+              best
+            </span>
+            Real estate
+            <a href="#" class="link">
+              here
+            </a>
+          </h1>
+          <div class="info__text">
+            <p>
+              <span class="info__selected-text info__selected-text--bold">
+                Over 50 years experience in love.
+              </span>
+              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p>
+              The market leader in real estate since 2003.<br>Awards Real Estate Company, 2010, 2011, 2013.<br>More than 200 satisfied customers premium segment.<br>We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p>
+              <span class="info__selected-text info__selected-text--bold">
+                We will never be late!<br>Guaranteed secure transaction - 100%.
+              </span>
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <h2 class="heading section__heading">
+          Gallery
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="gallery">
+          <div class="photos photos_wide_left">
```

Comment:
```suggestion
          <div class="photos photos_wide_left">
```
тут тоже не понятно что есть что. это блок и модификатор?
если да, то вот так:
```
          <div class="photos photos--wide-left">
```